### PR TITLE
Dynamic linking for logo from categories. 

### PR DESCRIPTION
**Pull request by codiejay**
When logo is clicked from categories or any other page it loads dynamically to the home page. 
closes https://github.com/EveripediaNetwork/issues/issues/797

### CODE REVIEW

**Review by Royal-lobster — APPROVED** [review]

**Comment by kesar** [comment]
its missing add proper linking to the categories in the homepage (that's why I took the screen) 👍🏻

**Comment by codiejay** [comment]
> its missing add proper linking to the categories in the homepage (that's why I took the screen) 👍🏻

Fixed this.

File: src/components/Landing/CategoriesList.tsx [hunk 1]
```diff
@@ -46,36 +46,35 @@ const CategoriesList = ({ categories }: CategoriesListProps) => {
...
-
-            <Text
-              py="4"
-              w="100%"
-              textAlign="center"
-              fontWeight="bold"
-              fontSize="lg"
-              size="md"
+          <Link href={`/categories/${category.id}`} key={category.id}>
+            <LinkBox
```
Comment (Royal-lobster) [inline src/components/Landing/CategoriesList.tsx:50]: @codiejay for this use LinkOverlay from the custom component in `components/elements/link`

we wrote custom components for all link items in that directory with chakra + next link

**Review by Royal-lobster — CHANGES_REQUESTED** [review]

File: src/components/Landing/CategoriesList.tsx (hunk 1 above)
Comment (codiejay) [reply src/components/Landing/CategoriesList.tsx:50]: There is a bug with the link overlay where it makes the whole page one big link. Will send you a demo privately.

File: src/components/Landing/CategoriesList.tsx [hunk 2]
```diff
@@ -47,34 +47,33 @@ const CategoriesList = ({ categories }: CategoriesListProps) => {
...
-              <LinkOverlay href={`/categories/${category.id}`}>
+              <Text
+                py="4"
+                w="100%"
+                textAlign="center"
+                fontWeight="bold"
+                fontSize="lg"
+                size="md"
+              >
                 {category.title}
```
Comment (Royal-lobster) [inline src/components/Landing/CategoriesList.tsx:74]: @codiejay you can wrap the `LinkOverlay` just for title, it links the entire card in `LinkBox`

File: src/components/Landing/CategoriesList.tsx [hunk 3]
```diff
@@ -47,34 +47,33 @@ const CategoriesList = ({ categories }: CategoriesListProps) => {
       >
         {categories.map(category => (
           <LinkBox
-            key={category.id}
```
Comment (Royal-lobster) [inline src/components/Landing/CategoriesList.tsx:50]: for key, you can just wrap the `LinkBox` with div and pass the key to that div. we can handle this with react forwardRef but this is simplier in this case.

**Review by kesar — APPROVED** [review]

**Pull request merged**